### ISSUE TITLE
CD is failing

### ISSUE THREAD

**Issue by dkmiller** [dkmiller/SqlHelper#3]
Travis deploy is failing ([example](https://travis-ci.org/dkmiller/SqlHelper/builds/353531644)) with:
```
Installing deploy dependencies
Successfully installed dpl-script-1.9.1
1 gem installed
!!! Script support is experimental !!!

Preparing deploy

Deploying application
./deploy.sh: 6: ./deploy.sh: nuget: not found
Script failed with status 127
failed to deploy
```

Take a look at [How do I deploy nuget packages in Travis CI?](https://stackoverflow.com/q/41509608/).

**Comment by dkmiller** [comment]
First install nuget (https://stackoverflow.com/a/43880251/). Don't need separate deploy script.

**Comment by dkmiller** [comment]
Most recent failed build: [18](https://travis-ci.org/dkmiller/SqlHelper/builds/353647997). Good news: the command "nuget push" was found. The bad news, 
```
System.IO.DirectoryNotFoundException: Could not find a part of the path '/home/travis/build/dkmiller/SqlHelper/SqlHelper/bin/Release'.
```
Got to figure out the correct local path to the `*.nupkg` file.

**Comment by dkmiller** [comment]
Fixed in #5
